### DISCUSSION TITLE
Support text alignement (justify, left, center, right)

### DISCUSSION THREAD

**Discussion by sy6sy2** [outline/outline#6589] [Ideas]
It should be interesting to be able to align the text in a paragraph?

**Comment by kylebial0** [comment]
+1 to this feature. I am shocked this is not a default feature that has already been implemented.

For an application where it's primary use is proving text documentation, this seems essential. There was a previous branch #6817  that seemed to accomplish this, but was closed by tommoor due to "Every feature [being a] maintenance burden for our tiny team"

**Reply by talha5389-teraception** [reply]
@tommoor I have a suggestion that can help have this feature (at least as workaround) without drawbacks discussed here https://github.com/outline/outline/pull/6817#issuecomment-2063807294. 

I currently can align text using single column table and row (its not elegant but works for me as I only need to do this in few places). I produces this result 

![image](https://github.com/user-attachments/assets/6c376212-7246-4245-8867-0cf5a3ea695a)

Only issue with this is that there is no way to disable table borders. Hiding/customization behavior table borders itself could be a potential feature as mentioned here https://github.com/outline/outline/discussions/7191#discussion-6894870


To overcome/workaround bug discussed in my comment here https://github.com/outline/outline/issues/6836#issuecomment-2405014399, I am using this workaround https://chatgpt.com/share/6707de5d-1d14-8002-99f6-db18312f45b8

**Reply by tommoor** [reply]
Are you using a super old version of the app or something? This issue is long resolved.

**Reply by talha5389-teraception** [reply]
I am using cloud hosted outline. This is my workspace url https://teraception1.getoutline.com/

**Comment by Maskil** [comment]
I wish it supports justified paragraphs because it looks way nicer with scientific documentations; I suppose it is rather a simple feature that toggles the alignment?

**Reply by TLRS-Sandwich** [reply]
If I'm reading the comments on #6817 correctly, I think that individual paragraphs can be aligned differently from others? Are you not able to do that in your version?

**Reply by Maskil** [reply]
Automatic alignment for languages that begins from right is implemented, but **manual** alignments are dropped in #6817 for complex maintenance. I'm asking for a toggle between justified text and standard left-aligned text, but it seems it falls into the same category of reasons for why it will never be implemented.

**Reply by TLRS-Sandwich** [reply]
That's so unfortunate, but maybe someday when they can get more funding.
